src/Native/MklProxyNative/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+project (MklProxyNative)
```

Comment:
>MklProxyNative [](start = 9, length = 14)

[https://github.com/dotnet/machinelearning/blob/1ae6070a554a374dec4d9b9f746cfde481b033c7/src/Common/AssemblyLoadingUtils.cs#L143](https://github.com/dotnet/machinelearning/blob/1ae6070a554a374dec4d9b9f746cfde481b033c7/src/Common/AssemblyLoadingUtils.cs#L143)
add it to the list.